src/main/webapp/jsp/instructorCourses.jsp

```diff
@@ -1,247 +1,34 @@
-                        <a class="btn btn-default btn-xs t_course_enroll<%=idx%>"
-                            href="<%=data.getInstructorCourseEnrollLink(courseDetails.course.id)%>"
-                            data-toggle="tooltip" data-placement="top" title="<%=Const.Tooltips.COURSE_ENROLL%>"
-                            <% if (!instructor.isAllowedForPrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_STUDENT)) {%>
-                                disabled="disabled"
-                            <% } %>
-                        >
-                            Enroll
-                        </a>
-                        <a class="btn btn-default btn-xs t_course_view<%=idx%>"
-                            href="<%=data.getInstructorCourseDetailsLink(courseDetails.course.id)%>"
-                            data-toggle="tooltip" data-placement="top" title="<%=Const.Tooltips.COURSE_DETAILS%>"
-                        >
-                            View
-                        </a>
-                        <a class="btn btn-default btn-xs t_course_edit<%=idx%>"
-                            href="<%=data.getInstructorCourseEditLink(courseDetails.course.id)%>"
-                            data-toggle="tooltip" data-placement="top" title="<%=Const.Tooltips.COURSE_EDIT%>"
-                        >
-                            Edit
-                        </a>
-                        <a class="btn btn-default btn-xs t_course_archive<%=idx%>"
-                            href="<%=data.getInstructorCourseArchiveLink(courseDetails.course.id, true, false)%>"
-                            data-toggle="tooltip" data-placement="top" title="<%=Const.Tooltips.COURSE_ARCHIVE%>"
-                        >
-                            Archive
-                        </a>
-                        <a class="btn btn-default btn-xs t_course_delete<%=idx%>"
-                            href="<%=data.getInstructorCourseDeleteLink(courseDetails.course.id,false)%>"
-                            onclick="return toggleDeleteCourseConfirmation('<%=courseDetails.course.id%>');"
-                            data-toggle="tooltip" data-placement="top" title="<%=Const.Tooltips.COURSE_DELETE%>"
-                            <% if (!instructor.isAllowedForPrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_COURSE)) { %>
-                                disabled="disabled"
-                            <% } %>
-                        >
-                            Delete
-                        </a>
-                    </td>
-                </tr>
-            <%      }
-                }
-                if (idx == -1){ // Print empty row
-            %>
-                <tr>
-                    <td></td>
-                    <td></td>
-                    <td></td>
-                    <td></td>
-                    <td></td>
-                    <td></td>
-                </tr>
-            <% } %>
-        </table>
+    </c:if>
```

Comment:
I think it shouldn't because putting it inside else clause will change the behaviour of the page.